src/util/PlayerMenu/PlayerMenu.ts

```diff
@@ -0,0 +1,185 @@
+                    value: "Hitting will get you a new card, be weary to not go over 21 or you lose!",
+                    inline: true,
+                },
+                {
+                    name: "Stay",
+                    value: "If you stay, you will stick with the hand that you've got until all players have stayed.",
+                    inline: true,
+                },
+            ]);
+
+        //If the player stays, we are going to remove the buttons and Give them a message to wait patiently
+        if (this.player.stay) {
+            embed
+                .setFields([])
+                .setImage(
+                    "https://media.discordapp.net/attachments/1090471775768428627/1099093963991961630/8IBKHtg0E484QKeXMPx4vyxD1czPK_ZzFtTQlMxm_c8.jpg?width=407&height=543"
+                )
+                .setDescription(
+                    `You have chosen to stay! \n\nWaiting for other players now! \n\nGood luck ${this.player.member.displayName}`
+                );
+        }
+
+        if (this.testForceStay) {
+            embed
+                .setFields([])
+                .setDescription(
+                    `You have been forced to stay with the hand you had because you took too long to choose. \n\nGood luck ${this.player.member.displayName}`
+                )
+                .setImage(
+                    "https://media.discordapp.net/attachments/1090471775768428627/1099093963991961630/8IBKHtg0E484QKeXMPx4vyxD1czPK_ZzFtTQlMxm_c8.jpg?width=407&height=543"
+                );
+        }
+
+        return [embed];
+    }
+
+    /**
+     * This will create the message to display to the play
+     * @param hideButtons This is to konw if we are hiding the buttons, only will happen if player stays
+     * @returns The object to display with the reply.
+     */
+    private generateMessagePayload(hideButtons: boolean) {
+        return {
+            embeds: this.generateEmbeds(),
+            components: hideButtons ? [] : [buttons],
+            ephemeral: true,
+        };
+    }
+
+    /**
+     * This function will intiialize the collector and reply depending on the buttons pressed.
+     */
+    private initCollector() {
+        if (!this.message) {
+            throw new Error(
+                "PlayerMenu error: Did not generate message correctly"
+            );
+        }
+
+        //initializer ofcollector
+        const collector = this.message.createMessageComponentCollector({
+            filter: (i) => i.user.id === this.interaction.user.id,
+            componentType: ComponentType.Button,
+            time: 10_000,
+        });
+
+        //The responses once buttons are pressed
+        collector.on("collect", (i) => {
+            i.deferUpdate();
+
+            if (i.customId === "stay") {
+                this.player.stay = true;
+                console.log("We stayed in ur mom");
+                this.render(true);
+                return;
+            } else {
```

Comment:
Else branch is not needed due to `return` in if() branch.
- Please redo console log messages.